x-pack/plugins/cases/public/components/all_cases/all_cases_generic.tsx

```diff
@@ -90,7 +94,7 @@ export const AllCasesGeneric = React.memo<AllCasesGenericProps>(
       setFilters,
       setQueryParams,
       setSelectedCases,
-    } = useGetCases();
+    } = useGetCases({ initialFilterOptions: { owner } });
```

Comment:
@jonathan-buttner @michaelolo24 Do you think is best for the `useGetCases` hook to use `useOwnerContext` instead of passing it as an argument?

Reply:
If you'll never call `useGetCases` without the context owner, then I say include it in the actual `useGetCases` call rather than passing it in, but if for some reason the `owner` can change from call to call, then pass it. There may be reasons I'm not thinking of, but that's how I work it out in my head